vocab/credentials/v2/vocabulary.yml

```diff
@@ -29,6 +29,12 @@ class:
     label: Credential status
     comment: A Credential Status provides enough information to determine the current status of the credential (for example, suspended or revoked). This class serves as a supertype for specific status types.
 
+  - id: ConfidenceMethod
+    label: Confidence method
+    defined_by: https://w3c-ccg.github.io/confidence-method-spec/
+    comment: A Confidence Method provides enough information for a verifier to determine whether the holder can generate a verifiable presentation to increase the verifier's confidence that they are the same entity referenced by the confidence method. This class serves as a supertype for specific status types.
```

Comment:
I need to work up some wording, and probably submit one or more PRs on the [`confidence-method-spec`](https://github.com/w3c-ccg/confidence-method-spec), but I feel strongly that ConfidenceMethod should apply to more than the specific given here; that ConfidenceMethod should be a generally applicable property that could be associated with any claim in a VC, and, similarly to Evidence, provide some basis for the issuer's claim(s) which is meant to increase the verifier's confidence in such claim(s) and/or issuer. In my mind, Evidence is non-repeatable, untestable, static information, while ConfidenceMethod is repeatable, testable, dynamic information.

Reply:
Well... I am just a messenger. I essentially copied extracts of text from the ccg document.

I would prefer _not_ do too much wordsmithing in this PR. If it is controversial, this should discussed on a dedicated issue/OR on confidence methods; after all, it is a "reserved" term for a reason. If you feel it is misleading, we can simply scrap the whole entry from the class definition, as there is a reference to the CCG document already.

Reply:
For the records: the detailed comment on the Confidence Method class has been removed and replaced by a boilerplate text that about a general superclass. The relevant discussion is to be done elsewhere, the conversation in this PR should be considered as resolved.